src/safeposix/syscalls/sys_calls.rs

```diff
@@ -201,11 +250,13 @@ impl Cage {
                 }
```

Comment:
In the above lines, we are again initializing the `sock_tmp` variable and incrementing the reference count of the Socket type Inode. Is it possible to reuse the above variables, or is there a significance behind this?

Reply:
Solved the bug